.changeset/perfect-jobs-remember.md

```diff
@@ -0,0 +1,6 @@
+---
+'@crowdstrike/glide-core': minor
+---
+
+- Checkbox Group's `value` is no longer reflected to match native.
+- Checkbox Group now removes from its `value` the `value` of a checked Checkbox that is disabled. This also works in reverse. If a disabled but checked Checkbox is enabled, it's `value` is added to Checkbox Group's `value`.
```

Comment:
Brought to Checkbox Group the [same](https://github.com/CrowdStrike/glide-core/pull/788) change I recently made to Dropdown Option. I'll do Radio Group next.